pkg/component/observability/monitoring/migration.go

```diff
@@ -53,8 +54,8 @@ type DataMigration struct {
 	ImageAlpine string
 	// StatefulSetName is the name of the old StatefulSet.
 	StatefulSetName string
-	// PVCName is the name of the old PersistentVolumeClaim.
-	PVCName string
+	// PVCNames is the list of the old PersistentVolumeClaims.
```

Comment:
Nit: Technically, `PVCNames` is the list of names of the old PersistentVolumeClaims.